cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -213,7 +213,7 @@
 
     {# TODO: Remove Else conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% if page and page.media %}
+    {% if page %}
```

Comment:
No, Im just wondering if we can get rid of the if statement and just have the for loop?